src/custom/pages/App/index.tsx

```diff
@@ -89,9 +88,6 @@ function createRedirectExternal(url: string) {
 const Loading = <LoadingWrapper>Loading...</LoadingWrapper>
 
 export default function App() {
-  // Dealing with empty URL queryParameters
-  useFilterEmptyQueryParams()
```

Comment:
Wouldn't it break anything else in the app?
I mean, it must have had a reason to be added, right?